packages/math-input/src/components/input/math-wrapper.ts

```diff
@@ -7,55 +7,57 @@
 import $ from "jquery";
 import MathQuill from "mathquill";
 
-import {DecimalSeparators} from "../../consts";
+import {DecimalSeparator} from "../../enums";
 import Keys from "../../data/keys";
 import {decimalSeparator} from "../../utils";
 
-import * as CursorContexts from "./cursor-contexts";
+import {CursorContext} from "./cursor-contexts";
 
 // Keeping `window` in place for test suite and GitHub Pages.
 // If it does not exist, fall back to CommonJS require. - jsatk
 
-const decimalSymbol = decimalSeparator === DecimalSeparators.COMMA ? "," : ".";
+const decimalSymbol = decimalSeparator === DecimalSeparator.COMMA ? "," : ".";
 
-const WRITE = "write";
-const CMD = "cmd";
-const KEYSTROKE = "keystroke";
-const MQ_END = 0;
+enum ActionType {
+    WRITE = "write",
+    CMD = "cmd",
+    KEYSTROKE = "keystroke",
+    MQ_END = 0,
+}
 
 // A mapping from keys that can be pressed on a keypad to the way in which
 // MathQuill should modify its input in response to that key-press. Any keys
 // that do not provide explicit actions (like the numeral keys) will merely
 // write their contents to MathQuill.
-const KeyActions = {
-    [Keys.PLUS]: {str: "+", fn: WRITE},
-    [Keys.MINUS]: {str: "-", fn: WRITE},
-    [Keys.NEGATIVE]: {str: "-", fn: WRITE},
-    [Keys.TIMES]: {str: "\\times", fn: WRITE},
-    [Keys.DIVIDE]: {str: "\\div", fn: WRITE},
+const KeyActions: {[K in Keys]?: {str: string; fn: ActionType}} = {
```

Comment:
We can't type this as `const KeyActions: Record<Keys: ...>` because it requires that we have an entry for every member of `Keys` then.